samples/snippets/src/test/java/com/example/spanner/SpannerSampleIT.java

```diff
@@ -51,12 +52,14 @@
 import org.junit.Assert;
 import org.junit.BeforeClass;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 /** Unit tests for {@code SpannerSample} */
 @RunWith(JUnit4.class)
 @SuppressWarnings("checkstyle:abbreviationaswordinname")
+@Category(SlowTest.class)
```

Comment:
These two tests were taking almost an hour. Plan is to introduce a job for slow sample integration tests(Java 11) to speed up the performance.

Reply:
Should we mark all the tests in SpannerSample as slow, or particualr backup/restore should be tagged?

Reply:
For now, I have marked both the files to not to run given that it's taking an almost an hour. and we are running these tests for each module. Goal is to run only once either with [install-without-bom] or [snapshot] or [snippets]